interpreter/text/lexer.mll

```diff
@@ -65,12 +65,15 @@ let floatop t f32 f64 =
   | "f64" -> f64
   | _ -> assert false
 
-let numop t i32 i64 f32 f64 =
+let fail_v128 = (fun s -> assert false)
```

Comment:
Does it show function name in the error message? If not you can try something like `failwith "<some message>"`, would be easier to see if you are hitting one of those cases or something else.

Reply:
Removed this in the latest commit after splitting v128.const into its own keyword, thanks!